src/road_gui.cpp

```diff
@@ -1129,8 +1129,13 @@ struct BuildRoadStationWindow : public PickerWindowBase {
 
 	void CheckOrientationValid()
 	{
-		if (_roadstop_gui_settings.orientation >= DIAGDIR_END) return;
+		this->RaiseWidget(WID_BROS_STATION_NE + _roadstop_gui_settings.orientation);
```

Comment:
I assume raising and then lowering the widget has to do with setting the displayed plane...but this could use a comment explaining what's going on for future readers. 🙂